requirements.txt

```diff
@@ -31,6 +31,7 @@ elasticsearch==1.3.0
 google-api-python-client==1.2
 python-crontab==1.9.2
 Babel==1.3
+schema==0.3.1
```

Comment:
Is this only used for testing? If so, than this can be moved to `requirements/dev.txt`.